wiki/beginners.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Beginners
+parent: Wiki
+---
+
+This page was moved from the old wiki and is in the process of being revised.
+{: .fs-5 .ls-10 .code-example }
+
+## Basic information
+
+When in doubt, consult the [API
+docs](https://remotestorage.github.io/remotestorage.js/files/remotestorage-js.html),
+or ask on IRC (\#remotestorage on Freenode), or on [the
+forums](https://community.remotestorage.io/).
+
+...
+
+## Get started
+
+### Choose a build
+
+There are a number of different builds available.
+
+#### Standard
+
+Files `remotestorage.js` and `remotestorage.min.js` (minified version).
+
+Contains all components of remotestorage.js for running in a browser.
+
+#### Without local cache
+
+Files `remotestorage-nocache.js` and `remotestorage-nocache.min.js`.
+
+Versions without any caching features included. Use this if you want
+your app to write directly to the remote server without caching any data
+in the browser’s storage (localStorage or IndexedDB).
+
+*Hint: Even if you "disable" or not enable caching for paths in the
+normal build, it will still cache outgoing changes. The only way to make
+sure something landed on the server immediately is to use this no-cache
+build.*
+
+#### AMD
+
+Files `remotestorage.amd.js` (with cache) and
+`remotestorage-nocache.amd.js` (without cache, see
+[below](#Without_local_cache "wikilink")).
+
+remoteStorage.js wrapped for use with
+[AMD](https://en.wikipedia.org/wiki/Asynchronous_module_definition)
+loaders such as [RequireJS](http://requirejs.org/).
+
+### Get remoteStorage.js
+
+#### Bower
+
+`bower install -S remotestorage`
+
+#### npm
+
+`npm install --save remotestoragejs`
+
+For node.js apps, check out the [node.js
+howto](https://github.com/remotestorage/remotestorage.js/blob/master/doc/nodejs.md)
+for more information.
+
+##### Webpack
+
+If you downloaded the latest release using `npm install
+remotestoragejs`, you can use the following line in your code:
+
+``` javascript
+require('remotestoragejs/release/stable/remotestorage.amd.js');
+// window.RemoteStorage is now available
+```
+
+You also need to define a shim for `XMLHttpRequest` such that Webpack
+won't be looking for it. In your `webpack.config.js`:
+
+``` javascript
+var webpack = require("webpack");
+  
+module.exports = {
+  externals: [
+    {xmlhttprequest: "{XMLHttpRequest:XMLHttpRequest}"}
+  ],
+}
+```
+
+We know that this sort of usage is not very aesthetic and are working on
+fixing it.
+
+#### Git
+
+` git clone `<https://github.com/remotestorage/remotestorage.js.git>
```

Comment:
This looks a little weird in the rendered output. Consider using a code block as with the previous comments.

    git clone https://github.com/remotestorage/remotestorage.js.git